obs-studio-client/source/properties.hpp

```diff
@@ -79,7 +79,14 @@ struct TextProperty : Property {
 		MULTILINE,
 	};
 
+	enum class InfoType {
```

Comment:
it will be duplicate of InfoType in source/obs-property.hpp

Reply:
I just followed existing approach here.
For the enum
```
	enum class Type {
		DEFAULT,
		PASSWORD,
		MULTILINE,
	};
```
ot has a duplicate in `source/obs-property.hpp`
```
	enum class TextType : uint8_t {
		Default,
		Password,
		MultiLine,
	};
```

Reply:
Should I avoid a code duplication?

Reply:
yes. if its possible better to not not have duplicated types.